pkgs/development/python-modules/vllm/default.nix

```diff
@@ -145,7 +159,11 @@ buildPythonPackage rec {
       cupy
     ];
 
-  stdenv = if cudaSupport then cudaPackages.backendStdenv else stdenv_pkg;
+  dontUseCmakeConfigure=true;
+
+  pythonRelaxDeps = true;
+
+  stdenv = if cudaSupport then cudaPackages.backendStdenv else stdenv;
```

Comment:
shouldn't this be 
```
if cudaSupport then cudaPackages.backendStdenv else stdenv_pkg;
```
?

Reply:
Also remember that there's an alternative to `let ... in`, which is to rewrite the top-level args as `{ ... }@args:` and use `if cudaSupport then ... else args.stdenv`

Reply:
sorry this was a merge fail, and great idea, will switch to using `@args`